src/panels/lovelace/entity-rows/hui-cover-entity-row.ts

```diff
@@ -0,0 +1,82 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../components/hui-generic-entity-row.js";
+import "../../../components/ha-cover-controls.js";
+import "../../../components/ha-cover-tilt-controls.js";
+
+import { isTiltOnly } from "../../../util/cover-model.js";
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+
+class HuiCoverEntityRow extends LitElement implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: EntityConfig;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      _config: {},
+    };
+  }
+
+  public setConfig(config: EntityConfig): void {
+    if (!config) {
+      throw new Error("Configuration error");
+    }
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <hui-generic-entity-row
+        .hass=${this.hass}
+        .config=${this._config}
+      >
+      ${
+        this.hass.states[this._config.entity]
```

Comment:
Migrate this to use the new no-entity-row from #1946 

```ts
const stateObj = this.hass.states[this._config.entity];

if (!stateObj) {
  return html`<hui-error-entity-row .entity="${this._config.entity}"></hui-error-entity-row>`;
}
```

Reply:
Why exactly are we wanting to use this new error row instead of the generic row? We still need the generic row to render the label, regardless, so just curious what we gain.

Reply:
Eventually we will remove the error rendering from generic row.

Reply:
Hmm, added this in and is not rendering anything. See it in the inspector but nothing showing. Too tired to look now, but will check it out further tomorrow during lunch, hopefully.